pulseapi/settings.py

```diff
@@ -165,18 +167,19 @@
 #
 
 # we want to restrict API calls to domains we know
-CORS_ORIGIN_ALLOW_ALL = False
+if '*' in env('CORS_WHITELIST'):
+    CORS_ORIGIN_ALLOW_ALL = True
+else:
+    # and we want origin whitelisting
+    CORS_ORIGIN_WHITELIST = os.getenv(
+        'CORS_ORIGIN_WHITELIST',
+        'localhost:3000,localhost:8000,localhost:8080,test.example.com:8000,test.example.com:3000'
```

Comment:
can't this default be specified in an env file? Then we can just use `env()` to grab the setting from the environment.